product-web-app/service/jobRolesService.ts

```diff
@@ -16,4 +17,13 @@ export default class JobRoleService {
       throw new Error('Error jobRoleService');
     }
   }
+
+  async createJobRoles(addJobRole: AddJobRole): Promise<Number> {
+    try {
+      const response = await axios.post(`${this.API_URL}/admin/job-roles`, addJobRole);
+      return response.status;
+    } catch (e) {
+      throw new Error('Could not create job role');
```

Comment:
it might be good to pass or check why server failed in adding a job-role, was it validation or API server failure, or it's a duplicate?

Reply:
have you made any changes to the above?